components/suggest/src/store.rs

```diff
@@ -721,7 +804,10 @@ mod tests {
         store.ingest(SuggestIngestionConstraints::default())?;
 
         store.dbs()?.reader.read(|dao| {
-            assert_eq!(dao.get_meta::<u64>(LAST_INGEST_META_KEY)?, Some(15));
+            assert_eq!(
+                dao.get_meta::<u64>("last_quicksuggest_ingest_data")?,
```

Comment:
Nit: Just in case we change the key format in the future, let's not use the hardcoded keys in the tests, we can call `SuggestRecordType::last_ingest_meta_key()` instead with the associated type.